components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -206,6 +206,13 @@ public class SCIMUserManager implements UserManager {
     private static final String MAX_LIMIT_RESOURCE_TYPE_NAME = "response-max-limit-configurations";
     private static final String MAX_LIMIT_RESOURCE_NAME = "user-response-limit";
 
+     /*
+     This will be expanded as few other claims to be added
+     Assuming these claims are immutable and will not be deleted.
+      */
```

Comment:
well this doesn't give the right idea. What you have to say is this list includes claims that can get updated over scim patch or put requests which will be recognized as the certain identity management flow from the IAM product perspective. And when handling such flows using this claim set the assumption is these claims are immutable and they cannot be deleted from a user's profile and only updates are allowed
e.g account enable and disable managed via xxx claim

Reply:
Fixed ac2cf820d6ea9f31971e7e71f756ef815fcf8e78